cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -184,13 +167,20 @@ def _remove_docker_creds():
 
 
 @pytest.fixture(scope="module")
-def local_setup_provider(request, setup_provider_modscope, provider, appliance):
+def local_setup_provider(request, ssa_single_vm, setup_provider_modscope, provider, appliance):
```

Comment:
@sbulage: It's not possible to inherit `ssa_single_vm` to `local_setup_provider`. As per my understanding `local_setup_provider` setup provider for your testing. so, `ssa_singl_vm` fixture will run first without setup provider for you. Its bad idea to inherit second-class candidate to the first-class candidate. Thum rule is `appliance` and `provider` are the first-class candidate for our `Integration test`.